app/services/flow/flow_state_machine.rb

```diff
@@ -143,6 +149,16 @@ def call_optional_show_step(optional_step)
       false
     end
 
+    def step_indicator_params
+      return if !IdentityConfig.store.ial2_step_indicator_enabled
```

Comment:
I thought rubocop was gonna get cranky and force `unless` here but it didn't which is great, I think negation is much easier to read than `unless`